config/crds/serving_v1alpha1_knativeserving_crd.yaml

```diff
@@ -51,6 +51,19 @@ spec:
               description: A means to override the corresponding entries in the upstream
                 configmaps
               type: object
+            registry:
+              description: A means to override the registry for all knative images.
+              type: object
+              properties:
+                default:
+                  description: The default registry to use for all knative iamges.
```

Comment:
Is this a registry or an image reference template?

Reply:
Image reference template. Updated the comment.